### DISCUSSION TITLE
visibility option for cards / sections should support "AND / OR" conditions

### DISCUSSION THREAD

**Discussion by ChristophCaina** [home-assistant/frontend#21020] [Dashboard]
it would be great, if the visibility could support OR conditions.

For example:

visibilits
Entity A
OR
Entity B
greater 1

**Comment by karwosts** [comment]
? 
![image](https://github.com/home-assistant/frontend/assets/32912880/0895ef50-9fca-49c2-ba4a-05f9008339b6)

**Reply by ChristophCaina** [reply]
well... let's say it that way:
It seems that I was far away on another topic with my thoughts... Thanks :-(
